src/oc/web/components/ui/site_header.cljs

```diff
@@ -3,6 +3,7 @@
    and every change here should be reflected there and vice versa."
   (:require [rum.core :as rum]
             [dommy.core :as dommy :refer-macros (sel1)]
+            [org.martinklepsch.derivatives :as drv]
```

Comment:
Not sure why I am getting this locally?
On staging I get the same but it is `app.main.css`

![screenshot](https://user-images.githubusercontent.com/55220/33270831-5c42df80-d353-11e7-91ae-bab341199588.png)